core/templates/pages/contributor-dashboard-page/services/contribution-and-review-backend-api.service.spec.ts

```diff
@@ -149,11 +149,11 @@ describe('Contribution and review backend API service', () => {
     it('should fetch reviewable suggestions from exp1', fakeAsync(() => {
       spyOn(carbas, 'fetchReviewableSuggestionsAsync').and.callThrough();
       const url = '/getreviewablesuggestions/exploration/translate_content' +
-      '?limit=10&offset=0&sort_key=Date&exploration_id=exp1';
+      '?offset=0&sort_key=Date&exploration_id=exp1';
 
       carbas.fetchSuggestionsAsync(
         'REVIEWABLE_TRANSLATION_SUGGESTIONS',
-        AppConstants.OPPORTUNITIES_PAGE_SIZE,
+        null,
```

Comment:
Also, why does this have to be `null`?

Reply:
In order to test if `fetchReviewableSuggestionsAsync` was called with null parameter when exploration_id is provided.

In CD dashboard, when exp_id is null, we will be fetching in batches like we do at present. And the feature this PR introduces sorts translation suggestions, which needs all the cards from backend to be fetched at one go. (Also reviewable cards are fetched when exp_id is not null)

Reply:
I see, thanks!